common/zoneops.go

```diff
@@ -180,30 +168,39 @@ func (mdb *MusicDB) DeleteZone(z *Zone) (error, string) {
 func (z *Zone) SetStopReason(tx *sql.Tx, value string) (error, string) {
 	mdb := z.MusicDB
 
-	localtx, tx, err := mdb.StartTransaction(tx)
-	if err != nil {
-		log.Printf("ZoneJoinGroup: Error from mdb.StartTransaction(): %v\n", err)
-		return err, "fail"
-	}
-	defer mdb.CloseTransaction(localtx, tx, err)
+//	localtx, tx, err := mdb.StartTransaction(tx)
```

Comment:
Can these comments be removed instead?

Reply:
Jag är på väg dit. Har missat på vissa ställen.

Reply:
Nice!